utils.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime, timedelta
+
+# Function to check valid end range
+
+
+def get_formatted_datetime(date_to_format: datetime) -> str:
+    """Convert a datetime object to a different string format
+
+    Arguments:
+        date_to_format {datetime} -- A datetime object
+
+    Returns:
+        str -- A date in the form of mm-dd-yyyy (e.g. 01-01-2020)
+    """
+    datetime_format = '%m-%d-%Y'
```

Comment:
something worth considering will be if you want to make the datetime_format a constant so you can use it in the function below thus not having to keep track of datetime formats in multiple places